Update the note for v12 of Storage SDK (#314)
MicrosoftDocs/dataexplorer-docs: data-explorer/kusto/api/netfx/kusto-ingest-client-rest.md

Finally, post the message that you constructed, to the selected ingestion queue that you obtained from Azure Data Explorer.

> [!NOTE]

> > .Net storage client before v12, when used, encodes the message to base64 by default. For more information, see [storage docs](https://docs.microsoft.com/dotnet/api/microsoft.azure.storage.queue.cloudqueue.encodemessage?view=azure-dotnet-legacy#Microsoft_WindowsAzure_Storage_Queue_CloudQueue_EncodeMessage).

Reviewer: ```suggestion
> .Net storage client versions below v12, when used, encode the message to base64 by default. For more information, see [storage docs](https://docs.microsoft.com/dotnet/api/microsoft.azure.storage.queue.cloudqueue.encodemessage?view=azure-dotnet-legacy#Microsoft_WindowsAzure_Storage_Queue_CloudQueue_EncodeMessage).
```

> > .Net storage client versions below v12, when used, encode the message to base64 by default. For more information, see [storage docs](https://docs.microsoft.com/dotnet/api/microsoft.azure.storage.queue.cloudqueue.encodemessage?view=azure-dotnet-legacy#Microsoft_WindowsAzure_Storage_Queue_CloudQueue_EncodeMessage).
> If you are NOT using that client (e.g. using v12 and above of Storage Client), make sure to properly encode the message content.

Reviewer 2: ```suggestion
> .Net storage client versions below v12, by default, encode the message to base64 For more information, see [storage docs](https://docs.microsoft.com/dotnet/api/microsoft.azure.storage.queue.cloudqueue.encodemessage?view=azure-dotnet-legacy#Microsoft_WindowsAzure_Storage_Queue_CloudQueue_EncodeMessage).
If you are using .Net storage client versions above v12, you must properly encode the message content.
```
